src/internal/connector/discovery/api/users.go

```diff
@@ -34,7 +35,17 @@ type Users struct {
 // ---------------------------------------------------------------------------
 
 type UserInfo struct {
-	DiscoveredServices map[path.ServiceType]struct{}
+	DiscoveredServices         map[path.ServiceType]struct{}
+	Purpose                    string
+	ArchiveFolder              string
+	Status                     string
+	DateFormat                 string
+	TimeFormat                 string
+	DelegateMeetMsgDeliveryOpt string
+	Timezone                   string
+	HasMailBox                 bool
+	HasOneDrive                bool
+	ErrGetMailBoxSetting       string
```

Comment:
Since these are all mailbox settings, it might be more maintainable for the future to construct a separate struct to hold them, like so:
```go
type mailboxInfo struct {

	Purpose                    string
	ArchiveFolder              string
	Status                     string
	DateFormat                 string
	TimeFormat                 string
	DelegateMeetMsgDeliveryOpt string
	Timezone                   string
	HasMailBox                 bool
	HasOneDrive                bool
	ErrGetMailBoxSetting       string
}
```

and then add that item to the userInfo.  That way you don't need to worry about future naming collisions, and more generic variables (Purpose, Status, Timezone) are already known to be part of the mailbox info, and not necessarily the user info at large (which may conflict across applications).
```suggestion
  mailbox mailboxInfo
```